hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/Import.java

```diff
@@ -413,7 +417,7 @@ public void map(ImmutableBytesWritable row, Result value, Context context) throw
             // skip if we filtered it out
             if (kv == null) continue;
             Cell ret = convertKv(kv, cfRenameMap);
-            context.write(new CellWritableComparable(ret), ret);
+            context.write(new CellWritableComparable(ret), new MapReduceExtendedCell(ret));
           }
```

Comment:
Can you please add a UT?

Reply:
Thanks for looking into this issue.
Comment addressed, please verify.